connector/src/main/java/com/google/cloud/spark/bigquery/examples/JavaShakespeare.java

```diff
@@ -28,16 +29,38 @@ public static void main(String[] args) {
     // by the connector. This assumes the Cloud Storage connector for
     // Hadoop is configured.
     String bucket = spark.sparkContext().hadoopConfiguration().get("fs.gs.system.bucket");
+    String inputBigqueryTable = "bigquery-public-data.samples.shakespeare";
+    String outputBigqueryTable = "wordcount_dataset.wordcount_output";
+
+    for (String arg : args) {
+      if (arg.charAt(0) == '-') {
+        switch (arg.charAt(1)) {
+          case 'b':
+            bucket = arg.substring(2);
+            break;
+          case 'i':
+            inputBigqueryTable = arg.substring(2);
+            break;
+          case 'o':
+            outputBigqueryTable = arg.substring(2);
+            break;
+          default:
+            usage();
+        }
+      }
+    }
```

Comment:
I think it will be simpler just to hard code positional parameters:

```java
inputBigqueryTable = arg[0];
outputBigqueryTable = arg[1];
```

Also, probably we do not need to parametrize input table, because it's publicly accessible anyway, so parametrization could be as simple as:

```java
String outputBigqueryTable = arg[0];
```

Reply:
Done